test/e2e/suites/import-gitops-v3/import_gitops_v3_test.go

```diff
@@ -157,3 +157,36 @@ var _ = Describe("[AWS] [EKS] - [management.cattle.io/v3] Create and delete CAPI
 		}
 	})
 })
+
+var _ = Describe("[GCP] [GKE] - [management.cattle.io/v3] Create and delete CAPI cluster functionality should work with namespace auto-import", Label(e2e.FullTestLabel), func() {
```

Comment:
Non-blocking question: if I am correct, new additions to e2e is not tested (full test running only in nightly tests) in the current PR, have you tried it running maybe locally?

Reply:
Yes, that is correct. There's no way to test this on PR tests as accessing secrets is not allowed for security reasons. It passes locally and I hope there's no issue when running in GitHub Actions, but I'll keep an eye on it after merging and try to spot any errors.

Reply:
Thanks for confirming @salasberryfin.